Simplified Login Panel

molongjoe opened the pull request:
Simplified Login Panel, Added Status Panel, Added Tabbing and Autologin Toggle

molongjoe committed:
Simplified Login Panel, Added Status Panel

molongjoe committed:
Added tabbing and autologin toggle

ndru-pi-squared commented:
Real perrty, works well. Only suggestion I have is that when the user clicks "Login Different User" we should change the status panel text back to "Waiting for user to do something..." I'll look into it. Not a big deal at all but a minor change that we can add in whenever really

ndru-pi-squared committed:
CameraAnimationHandler.MoveRight() --> now resets the stats panel text

I also added a namespace to PlayFabController, and added a null check for username and password (and status panel text to accompany it).

I tried to "break" this via monkey testing and was genuinely having trouble getting it to produce an unhanded error or undefined behavior.

I also started modifying the error messages to prevent sensitive data exposure. Not 100% sure why all of them are not being replaced atm but there are more pressing things to jump into now.

ndru-pi-squared committed:
Merge branch 'develop' into BM-LoginFix
